### ISSUE TITLE
allow scoring callbacks to work with sklearn metrics

### ISSUE THREAD

**Issue by benjamin-work** [dnouri/skorch#174]
The scoring callbacks don't accept sklearn metrics as scoring function (e.g. `sklearn.metrics.accuracy`). Maybe we should make this possible by internally calling `sklearn.metrics.make_scorer` if we detect this (if that is possible).

This requires to resolve #173 first.
